kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/templates/testclients/TestClientsJobTemplates.java

```diff
@@ -78,7 +78,7 @@ public static JobBuilder defaultAdminClientJob(String jobName, List<String> args
                 .withContainers(new ContainerBuilder()
                         .withName("admin")
                         .withImage(Constants.TEST_CLIENTS_IMAGE)
-                        .withImagePullPolicy(Constants.PULL_IMAGE_IF_NOT_PRESENT)
+                        .withImagePullPolicy(Constants.PULL_IMAGE_ALWAYS)
```

Comment:
This is certainly the simplest thing that could work, but its going to slow down all the tests as they re-pull images that haven't changed. Thats especially true for the Kcat and KAF images which are stable release tags. Maybe we should make it so we can customise the pull policy by client (or maybe we don't change it here and just override the pull policy in the StrimziClient driver?)

Reply:
kcat and kaf don't go via these methods

Reply:
there's methods `defaultKcatJob` and `defaultKafkaGoConsumerJob`. 

`newJobForContainer` might be a risky one to have it set as default since it's a general method something else could use.

Reply:
Oh is `testclients` the strimzi only package? I was guessing it supported all clients. In that case lets leave it as it is.

Reply:
Yes, test clients only supports Java Client kafka